src/main/java/mate/academy/springboot/datajpa/dto/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+import mate.academy.springboot.datajpa.dto.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CategoryMapper {
+    public CategoryResponseDto toResponseDto(Category category) {
+        CategoryResponseDto categoryResponseDto = new CategoryResponseDto();
+        categoryResponseDto.setId(category.getId());
+        categoryResponseDto.setName(categoryResponseDto.getName());
```

Comment:
You are setting the name of the CategoryResponseDto with its own name. You need to set it with the name from Category model.